client/app/account/user/translations/Messages_fr_FR.xml

```diff
@@ -626,7 +626,7 @@
    <translation id="user_account_security_double_auth_wizard_sms_send" qtlid="52807">Envoyer</translation>
    <translation id="user_account_security_double_auth_wizard_u2f_add_success" qtlid="441164">La clé a été ajoutée avec succès.</translation>
    <translation id="user_account_security_double_auth_wizard_congratulation_title" qtlid="441177">Bravo !</translation>
-   <translation id="user_account_security_double_auth_wizard_congratulation_info1" qtlid="441190">Votre compte est maintenant sécurisé !</translation>
+   <translation id="user_account_security_double_auth_wizard_congratulation_info1">Votre compte est maintenant sécurisé!</translation>
```

Comment:
Sounds useless. (You must have a space before `!`).

Reply:
But that will cause the word to wrap to the next line. Do you think we will have to leave it that way?

Reply:
This has been fixed by using a new css class